kiosk/next.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   reactStrictMode: true,
   images: {
-    domains: ['demoedge.sitecoresandbox.cloud'],
-},
-}
+    domains: ['demoedge.sitecoresandbox.cloud', 'playsummit.sitecoresandbox.cloud'],
```

Comment:
Please also add it to `Website\src\rendering\next.config.js` if it is not there already.